stackslib/src/chainstate/stacks/boot/pox-4.clar

```diff
@@ -58,6 +58,33 @@
 (define-data-var configured bool false)
 (define-data-var first-pox-4-reward-cycle uint u0)
 
+;; PoX mainnet constants
+;; Min/max number of reward cycles uSTX can be locked for
+(define-constant MIN_POX_REWARD_CYCLES u1)
+(define-constant MAX_POX_REWARD_CYCLES u12)
+
+;; Default length of the PoX registration window, in burnchain blocks.
+(define-constant PREPARE_CYCLE_LENGTH (if is-in-mainnet u100 u50))
+
+;; Default length of the PoX reward cycle, in burnchain blocks.
+(define-constant REWARD_CYCLE_LENGTH (if is-in-mainnet u2100 u1050))
+
+;; Stacking thresholds
+(define-constant STACKING_THRESHOLD_25 (if is-in-mainnet u20000 u8000))
+
+;; MOCK
+;; Allow to set stx-account details for any user
+;; These values are used for PoX only
+(define-map mock-stx-account-details principal {unlocked: uint, locked: uint, unlock-height: uint})
+
+(define-read-only (get-stx-account (user principal))
+    (default-to (stx-account user) (map-get? mock-stx-account-details user)))
+
+(define-public (mock-set-stx-account (user principal) (details {unlocked: uint, locked: uint, unlock-height: uint}))
```

Comment:
We can't have mocking in production

Reply:
Yeap. Understood. When you have a minute let's hop on a meeting to discuss what we can / can't bring over from your previous clar-unit branch & the sbtc-mini branch. 

Looking at those again I noticed we made 'mock' more functions than I remembered in sbtc-stacking pool.

What should / shouldn't we mock/setup here to be able to test flow tests?

Reply:
I like the concept that we only test what we can test here. Maybe clarinet can add mocking 

Here are more candidates for clar tests once https://github.com/stacks-network/stacks-core/pull/4157 is merged:
https://github.com/stacks-network/stacks-core/blob/92010c4c226ab59b9ab8867162c71baa1452f9fc/contrib/core-contract-tests/tests/pox/check_revoke_delegate_stx.test.ts